sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/FailoverTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Net;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class FailoverTests
+    {
+        [Fact]
+        public async void FailoverDatabase()
+        {
+            using (SqlManagementTestContext context = new SqlManagementTestContext(this))
+            {
+                ResourceGroup resourceGroup = context.CreateResourceGroup();
+                SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+                Server server = context.CreateServer(resourceGroup);
+
+                // Create database
+                string dbName = SqlManagementTestUtilities.GenerateName();
+                var dbInput = new Database()
+                {
+                    Location = server.Location
+                };
+                var db = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName, dbInput);
+                Assert.NotNull(db);
+
+                // Failover database
+                AzureOperationResponse failoverResponse = await sqlClient.Databases.BeginFailoverWithHttpMessagesAsync(
```

Comment:
just call `Failover` instead, that way you are verifying the entire e2e operation succeeds.